frontend/src/reducers/roomSlice.js

```diff
@@ -0,0 +1,65 @@
+/* eslint-disable no-param-reassign */
+import { createSlice } from '@reduxjs/toolkit';
+import { ROLES } from '../utils/roles';
+
+export const roomSlice = createSlice({
+  name: 'currentRoom',
+  initialState: {
+    id: null,
+    participants: [],
+  },
+  reducers: {
```

Comment:
I'm not very familiar with Redux. Since room ID, list of participants and role change for each room, and don't really make sense app-wide, is it still useful to keep this info in the store? Couldn't we just keep this as state variables inside the `Room` view?

Just trying to understand the benefits of each approach :)

Reply:
I used the store just to avoid having a ref for the state of the participants (just using the state doesn´t work since we are working with events and the state gets "outdated"). By using the store + reducers, we can have one common state that gets updated by those dispatchers, without the need of refs. 

Also I saw it useful for the room-admin panel that will probably need some data from the store 🙌